packages/@react-spectrum/s2/chromatic/DatePicker.stories.tsx

```diff
@@ -0,0 +1,111 @@
+/*
+ * Copyright 2024 Adobe. All rights reserved.
+ * This file is licensed to you under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License. You may obtain a copy
+ * of the License at http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under
+ * the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+ * OF ANY KIND, either express or implied. See the License for the specific language
+ * governing permissions and limitations under the License.
+ */
+
+import {CalendarDate, CalendarDateTime} from '@internationalized/date';
+import {DatePicker} from '../';
+import type {Meta, StoryObj} from '@storybook/react';
+import {userEvent} from '@storybook/test';
+
+const meta: Meta<typeof DatePicker> = {
+  component: DatePicker,
+  parameters: {
+    chromaticProvider: {disableAnimations: true}
+  },
+  title: 'S2 Chromatic/DatePicker'
+};
+
+export default meta;
+
+type Story = StoryObj<typeof DatePicker>;
+
+const date = new CalendarDate(2022, 2, 3);
+
+export const Default: Story = {
```

Comment:
Maybe a good time to bring up with Spectrum whether we would want the height to be static so we don't get the "jumping" behavior when switching between months with different numbers of rows. Seems like a UX improvement particularly for `labelPosition="top"`